config/jobs/image-pushing/k8s-staging-sig-docs.yaml

```diff
@@ -6,7 +6,6 @@ postsubmits:
         testgrid-dashboards: sig-docs-website, sig-k8s-infra-gcb
         testgrid-tab-name: post-website-push-image-k8s-website-hugo
       decorate: true
-      run_if_changed: "^Dockerfile$"
```

Comment:
I think it's OK to only run on a change to one of:
- `Dockerfile`
- `Makefile`
- `netlify.toml`

Reply:
I'm not sure which flavor of regex this is, but I presume we / you could revise the match to cover any of those 3?

Reply:
We can do that as a followup PR.